demo/css/main.css

```diff
@@ -34,3 +34,7 @@ body {
 .dialog.is-active {
     display: block !important;
 }
+
+.is-no-scroll {
```

Comment:
Not sure if it makes sense to introduce the concept of helpers in this library's CSS and I don't think there's gonna be any reuse of this anyway. I think a state hook makes sense for this.